commcare_connect/microplanning/tests/test_views.py

```
@@ -1940,73 +1974,95 @@ def _get_metric(self, metrics, name):
-        m = self._get_metric(metrics, "Days Remaining")
-        assert m["value"] == 5
-        assert "percentage" not in m
-
     def test_unvisited_count_and_percentage(self, opp):
-        """Unvisited = WAs with 0 approved visits, among non-excluded.
-
-        Inaccessible with 0 approved visits is included.
-        """
-        wa_visited, wa_pending_only, wa_empty, wa_inaccessible, wa_excluded = self._make_work_areas(
+        """Unvisited = WAs still carrying the NOT_VISITED status, among in-scope areas."""
+        wa_not_visited, wa_visited, wa_inaccessible, wa_excluded = self._make_work_areas(
```

Ruff (0.16.1)

[warning] Unpacked variable `wa_not_visited` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)